### PR TITLE
Add events: Milestone 3

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this milestone, I did the following:

- Copied the files calculator.js and operate.js into a logic/ directory in my project.
- Analyzed the files calculator.js and operate.js, to be sure that I understand what they do.
- Imported the files in my Calculator component specifically in the calculator.js file
- Implemented the event handlers using the math logic from calculator.js and operate.js in my React component.
- Made use of state and props, pass props to child components, use lifecycle methods, lift state up.
- Tested my app in the browser to make sure that it performs math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,76 +1,180 @@
...
             %
           </button>
-          <button type="button" className="right-most cell4">
+          <button
+            name="/"
+            type="button"
+            className="right-most cell4"
+            onClick={(ev) => this.handleClick(ev.target.name)}
+          >
             /
```
Comment (DeliceLydia) [inline src/components/Calculator.js:58]: - [ ] Please work on the functionality of this event, currently when I try to use `/` I get this error
div

N.B: Remember that when you divide a number by `zero` let us say `5/0`
you are supposed to get `error` in the screen

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @pasytchangwa , 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (pasytchangwa) [reply src/components/Calculator.js]: Thank you for the review. I have made the necessary changes

Comment (henatan99) [reply src/components/Calculator.js]: - This issue is well addressed! :+1: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @pasytchangwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**